### PR TITLE
refactor(core): Migrate from crypto-js to native crypto

### PR DESCRIPTION
**Pull request by netroy**
[`crypto-js` has been discontinued](https://github.com/brix/crypto-js/commit/1da3dabf93f0a0435c47627d6f171ad25f452012)

PS: We'll remove `crypto-js` usage from `n8n-workflow` and `@n8n_io/license-sdk` in separate PRs.

### CODE REVIEW

File: packages/core/src/Cipher.ts [hunk 1]
```diff
@@ -1,21 +1,43 @@
 import { Service } from 'typedi';
-import { AES, enc } from 'crypto-js';
+import { createHash, createCipheriv, createDecipheriv, randomBytes } from 'crypto';
```
Comment (valya) [inline packages/core/src/Cipher.ts:2]: Maybe not for this commit but do you think we should be moving the code base to the `node:` namespace node imports?

**Review by valya — APPROVED** [review]
:+1:

File: packages/core/src/Cipher.ts (hunk 1 above)
Comment (netroy) [reply packages/core/src/Cipher.ts:2]: yeah. I think we should migrate all node imports in a single PR at some point.

**Comment by netroy** [comment]
Let's merge this **after** today's release.

**Comment by janober** [comment]
Got released with [`n8n@1.16.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.16.0)

**Pull request merged**